src/components/Auth/SignupOption.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { theme } from 'styles/theme';
+
+interface SignupOptionProps {
+  type: string;
```

Comment:
```suggestion
  type: 'nickname' | 'emailConfirm' | 'password';
```

정확한 타입을 제공할 수 있을 것 같네요~